examples/filetimes.sh

```diff
@@ -19,8 +14,8 @@ ${timer} python filetimes.py ${1}.parq        dask    census meterswest metersno
 ${timer} python filetimes.py ${1}.snappy.parq dask    census meterswest metersnorth race
 ${timer} python filetimes.py ${1}.parq        pandas  census meterswest metersnorth race
 ${timer} python filetimes.py ${1}.snappy.parq pandas  census meterswest metersnorth race
-${timer} python filetimes.py ${1}.castra      dask    census meterswest metersnorth race
-#${timer} python filetimes.py ${1}.castra      pandas  census meterswest metersnorth race
+#${timer} python filetimes.py ${1}.castra      dask    census meterswest metersnorth race
+${timer} python filetimes.py ${1}.castra      pandas  census meterswest metersnorth race
```

Comment:
I get `AttributeError: module 'dask.dataframe' has no attribute 'from_castra'` from filetimes.py. I'm pushing up a commit shortly that simply comments it out - perhaps that's a better approach.